api/src/pcapi/utils/login_manager.py

```diff
@@ -24,19 +24,25 @@ def get_request_authorization() -> werkzeug.datastructures.Authorization | None:
 
 
 @app.login_manager.user_loader  # type: ignore [attr-defined]
-def get_user_with_id(user_id: int) -> users_models.User | None:
+def get_user_with_id(user_id: str) -> users_models.User | None:
     flask.session.permanent = True
     session_uuid = flask.session.get("session_uuid")
     if not users_models.UserSession.query.filter_by(userId=user_id, uuid=session_uuid).one_or_none():
         return None
 
     try:
         if flask.request.blueprint.startswith(backoffice_web.name):  # type: ignore [union-attr]
-            return backoffice_api.fetch_user_with_profile(user_id)
+            return backoffice_api.fetch_user_with_profile(int(user_id))
     except AttributeError:
         pass
 
-    return users_models.User.query.filter(users_models.User.id == user_id).one_or_none()
+    internal_admin_id = flask.session.get("internal_admin_id", 0)
+    user = users_models.User.query.filter(users_models.User.id == user_id).one_or_none()
```

Comment:
Détail : je trouve ça plus lisible dans l'ordre : 
```suggestion
    user = users_models.User.query.filter(users_models.User.id == user_id).one_or_none()
    internal_admin_id = flask.session.get("internal_admin_id", 0)
```